tests/transforms/test_indices.py

```diff
@@ -7,7 +7,16 @@
 import torch
 from torch import Tensor
 
-from torchgeo.transforms import indices
+from torchgeo.transforms import (
+    AppendGNDVI,
+    AppendNBR,
+    AppendNDBI,
+    AppendNDSI,
+    AppendNDVI,
+    AppendNDWI,
+    AppendSWI,
+    indices,
```

Comment:
Could replace this with `AppendNormalizedDifferenceIndex` for consistency

Reply:
It took me a startling long time to figure out what you meant here